casepro/backend/junebug.py

```diff
@@ -115,7 +175,28 @@ def pull_contacts(
             tuple of the number of contacts created, updated, deleted and
             ignored
         """
-        return (0, 0, 0, 0)
+        identity_store = self.identity_store
+
+        # all identities created in the Identity Store in the time window
+        new_identities = identity_store.get_identities(
+            created_at__gte=modified_after)
```

Comment:
Wouldn't that mean we would get all existing identities? Since `modified_before` is probably the current time. Don't we instead want all the identities that were created since the last time we ran this?